### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by Mhdez221993**
# Project requirements

* Build your project schema.
  * Create and run the necessary migration files.
  * Table and column names should match the ERD diagram.
    * note: photo for users table should be a link to an image
  * Foreign keys should be included.
  * All coulms that are foreign keys should have a corresponding index.

# Thank you for reviewing my project 👍

### CODE REVIEW

File: db/migrate/20220120201803_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postcounter
+
```
Comment (Emmyn5600) [inline db/migrate/20220120201803_create_users.rb:8]: - [ ]  Optional suggestion

- I would recommend renaming/changing the columns like `posts_counter`

File: db/migrate/20220120201907_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
```
Comment (Emmyn5600) [inline db/migrate/20220120201907_create_posts.rb:7]: - [ ] Optional suggestion

- I would recommend renaming/changing the columns like `comments_counter` and `likes_counter`.

**Review by Emmyn5600 — APPROVED** [review]
Hi @Mhdez221993 

Woo ow 👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

- Check the comment under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: db/migrate/20220120201907_create_posts.rb (hunk 2 above)
Comment (Mhdez221993) [reply db/migrate/20220120201907_create_posts.rb:7]: Thank you for the suggestion 👍

**Pull request merged**